azkaban-common/src/main/java/azkaban/executor/Statistics.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.executor;
+
+import java.util.Date;
+import java.util.Map;
+
+  public class Statistics {
+    private double remainingMemoryPercent;
+    private long   remainingMemory;
+    private int    remainingFlowCapacity;
+    private int    numberOfAssignedFlows;
+    private Date   lastDispatchedTime;
+    private long   remainingStorage;
+    private double cpuUsage;
+    private int    priority;
```

Comment:
now I expose both the remainingFlowCapacity and the numberOfAssignedFlows which represents the percentage of flow capacity I think we are fine here,  I've already modified the comparator to use the numberOfAssignedFlows to judge the weight.